app/src/androidTest/java/org/mozilla/focus/activity/SettingsGeneralTest.kt

```diff
@@ -107,6 +108,7 @@ class SettingsGeneralTest {
         }
     }
 
+    @Ignore("Compose scrolling issue: https://github.com/mozilla-mobile/focus-android/issues/7282")
```

Comment:
I created an issue for the disabled test to keep track of it, if you wish to change the link here: https://github.com/mozilla-mobile/focus-android/issues/7718